collections/hub/golden_path/negative_scenarios/quotes_negative.json

```diff
@@ -1297,14 +1325,22 @@
                 "id": 5,
                 "description": "Response should contain payer",
                 "exec": [
-                  "expect(response.body.errorInformation.errorDescription).to.include('payer')"
+                  "if(environment.API_TYPE === 'fspiop') {",
+                  "  expect(response.body.errorInformation.errorDescription).to.include('payer')",
+                  "} else {",
+                  "  expect(response.body.errorInformation.errorDescription).to.include('/requestBody/CdtTrfTxInf must have required property \\'Dbtr\\'')",
+                  "}"
                 ]
               },
               {
                 "id": 6,
-                "description": "Response should contain '3101'",
+                "description": "Response should contain '3101' or 3102 ",
```

Comment:
lets log a bug for this, add link and move forward with RC testing; based on the outcome, we can proceed for the release whether to keep this as is or if we can arrive at the same code